tools/setup_perf_cluster.sh

```diff
@@ -238,6 +260,13 @@ function setup_all() {
   setup_cluster_all
 }
 
+function delete_all() {
+  delete_istio
```

Comment:
let’s add a confirmation message and sleep for a chance for the user to control C out first